
### Examining racial discrimination in the US job market

#### Background
Racial discrimination continues to be pervasive in cultures throughout the world. Researchers examined the level of racial discrimination in the United States labor market by randomly assigning identical résumés black-sounding or white-sounding names and observing the impact on requests for interviews from employers.

#### Data
In the dataset provided, each row represents a resume. The 'race' column has two values, 'b' and 'w', indicating black-sounding and white-sounding. The column 'call' has two values, 1 and 0, indicating whether the resume received a call from employers or not.

Note that the 'b' and 'w' values in race are assigned randomly to the resumes.

#### Exercise
You will perform a statistical analysis to establish whether race has a significant impact on the rate of callbacks for resumes.

Answer the following questions **in this notebook below and submit to your Github account**. 

   1. What test is appropriate for this problem? Does CLT apply?
   2. What are the null and alternate hypotheses?
   3. Compute margin of error, confidence interval, and p-value.
   4. Discuss statistical significance.

You can include written notes in notebook cells using Markdown: 
   - In the control panel at the top, choose Cell > Cell Type > Markdown
   - Markdown syntax: http://nestacms.com/docs/creating-content/markdown-cheat-sheet


#### Resources
+ Experiment information and data source: http://www.povertyactionlab.org/evaluation/discrimination-job-market-united-states
+ Scipy statistical methods: http://docs.scipy.org/doc/scipy/reference/stats.html 
+ Markdown syntax: http://nestacms.com/docs/creating-content/markdown-cheat-sheet

****

In [1]:
import pandas as pd
import numpy as np
from scipy import stats

In [68]:
data = pd.io.stata.read_stata('data/us_job_market_discrimination.dta')

In [69]:
# number of callbacks for balck-sounding names
sum(data[data.race=='b'].call)

157.0

In [70]:
sum(data[data.race=='w'].call)

235.0

(1) The z-test is appropriate.  There are a sufficent number of samples fot the centrol limit theorem to apply and we can treat the same like a normal distribution.

(2) The null hypotheses is "There is no differnece between the mean of the call rate between resumes with black sounding names and resumes with white sounding names".  The alternative hypothese is "There is a differnece between the mean of the call rate between resumes with black sounding names and resumes with white sounding names".

In [71]:
data2 = pd.DataFrame(data, columns = ["race", "call"])

In [72]:
data2[data2.race == 'w'].describe()

call
count  2435.000000
mean      0.096509
std       0.295346
min       0.000000
25%       0.000000
50%       0.000000
75%       0.000000
max       1.000000

In [73]:
data2[data2.race == 'b'].describe()


call
count  2435.000000
mean      0.064476
std       0.245649
min       0.000000
25%       0.000000
50%       0.000000
75%       0.000000
max       1.000000

In [74]:
mean_w = 0.096509
std_w = 0.295346
n_w = 2435
mean_b = 0.064476
std_b = 0.245649
n_b = 2435

In [75]:
mean_w, std_w, n_w, mean_b, std_b, n_b

(0.096509, 0.295346, 2435, 0.064476, 0.245649, 2435)

In [76]:
std_err = (std_w**2/n_w + std_b**2/n_b)**(0.5)

In [77]:
std_err

0.0077849085542443346

In [78]:
mean_difference = mean_w - mean_b

In [79]:
mean_difference

0.03203299999999999

In [80]:
confidence_interval_95_percent = (mean_difference - 2 * std_err, mean_difference + 2 * std_err)

In [81]:
confidence_interval_95_percent

(0.016463182891511323, 0.04760281710848866)

In [82]:
z = mean_difference/std_err

In [83]:
z

4.114756104943017

(3) 
The margin of error (std_err) is 0.008.
The 95% confidence interval of where the true difference in means is (0.016, 0.047).
With this high z-score of 4.1, the p-value is less than 0.0001

(4)
The is strong statistical evidence that white sounding names are called back more frequently.  The value of '0' is outsided the 95% confidence interval, and the p-value is very small indicating that the data we see is unlikely due to pure randomness.